src/utils/string-test.js

```diff
@@ -162,3 +163,19 @@ describe('swgTransactionId', () => {
     }
   });
 });
+
+describe('hash', async () => {
+  it('should create 128 character hexadecimal hashes', async () => {
+    let result = null;
+    await hash('string1').then((h) => (result = h));
+    expect(result.length).to.equal(128);
+  });
+
+  it('should create unique hashes', async () => {
+    let hash1 = null;
+    let hash2 = null;
```

Comment:
Please use the values of the `await` expressions to define these variables more succinctly. Ex: `const hash1 = await ...`